smoke_tests/src/test/java/uk/gov/dhsc/htbhf/smoke/ServiceIsRunningSpec.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.dhsc.htbhf.smoke;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.net.URI;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.springframework.http.HttpStatus.OK;
+
+@ExtendWith(SpringExtension.class)
+class ServiceIsRunningSpec {
```

Comment:
Can we call this `ServiceIsRunningTest` and not spec?

Reply:
Renamed